browser/main/lib/ConfigManager.js

```diff
@@ -58,6 +58,7 @@ export const DEFAULT_CONFIG = {
     delfaultStatus: 'PREVIEW', // 'PREVIEW', 'CODE'
     scrollPastEnd: false,
     type: 'SPLIT', // 'SPLIT', 'EDITOR_PREVIEW'
+    isStacking: false,
```

Comment:
I think this should be in `UI` instead of `editor`

Reply:
Done